buildroot-external/board/pc/ova/kernel.config

```diff
@@ -12,6 +12,16 @@ CONFIG_VMWARE_VMCI=y
 CONFIG_VMWARE_BALLOON=y
 CONFIG_VMWARE_PVSCSI=y
 
+# These options are for LXD Guest Support
+CONFIG_NET_9P=y
+CONFIG_NET_9P_VIRTIO=y
+CONFIG_9P_FS=y
+CONFIG_9P_FS_POSIX_ACL=y
```

Comment:
Only comment from my side: It is really necessary to integrate all these 9P config option directly into a kernel or isn't it better to compile them as kernel modules instead? IMHO, compiling them as loadable kernel modules should be a better approach.